sdk/appcenter/src/test/java/com/microsoft/appcenter/persistence/DatabasePersistenceTest.java

```diff
@@ -260,4 +260,19 @@ public Log answer(InvocationOnMock invocation) {
         /* Verify that the only log we deleted in the entire test was the one from previous test (id=1). */
         verify(databaseStorage).delete(anyLong());
     }
+
+    @Test
+    public void checkSetStorageSizeForwarding() {
```

Comment:
I think we can combine two tests.

Reply:
The `setMaxStorageSize` method is passed down through both the StorageHelper and DefaultChannel layers, and that code needs to be covered.  It makes sense to put the testing code in the test files for their respective class